lib/redirect.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+var url = require('url')
+var isUrl = /^https?:/
+
+function Redirect (request) {
+  this.request = request
+  this.followRedirect = true
+  this.followRedirects = true
+  this.followAllRedirects = false
+  this.allowRedirect = function () {return true}
```

Comment:
We probably shouldn't be defining this for every instance, given it just defaults to true.
You should probably define this has a named function and then assign this.allowRedirect to the function.